src/Ledger/Ratify.lagda

```diff
@@ -48,9 +126,18 @@ record RatifyState : Set where
         removed         : List (GovActionID × GovActionState)
         delay           : Bool
 \end{code}
+} %% end small
 \caption{Types and functions for the RATIFY transition system}
 \end{figure*}
-\begin{figure*}[h]
+As mentioned earlier, each \GovAction must include a \GovActionID
+for the most recently enacted action of its given type. Consequently, two actions of the
+same type can be enacted at the same time, but they must be \emph{deliberately}
+designed to do so.
+\begin{itemize}
```

Comment:
Yes, I see why it seems weird because the corresponding code shows up later in the PDF.  I'm going to add `!` to the `[h]` argument of the figures and maybe they'll be placed more appropriately.  I'll also add a sentence or clause before the list that connects it to the figure containing the corresponding code.